src/main/java/com/conveyal/r5/streets/LinkedPointSet.java

```diff
@@ -134,11 +133,11 @@ public LinkedPointSet (PointSet pointSet, StreetLayer streetLayer, StreetMode st
         // Null means relink and rebuild everything, but this will be constrained below if a base linkage was supplied.
         Geometry linkageCostRebuildZone = null;
 
+        // TODO general purpose method to check compatability
         if (baseLinkage != null && (
-                !(baseLinkage.pointSet == ((WebMercatorGridPointSet) pointSet).base ||
-                 baseLinkage.pointSet == pointSet) ||
+                baseLinkage.pointSet != pointSet) ||
```

Comment:
A test failed after this commit - maybe a stray paren on this line? Looks like the closing paren should be just before the opening curly brace of the conditional body.